go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/kisielk/errcheck v1.7.0
 	github.com/mattn/go-shellwords v1.0.12
 	github.com/opentracing/opentracing-go v1.2.0
-	github.com/stretchr/testify v1.9.0
+	github.com/stretchr/testify v1.10.0
```

Comment:
Is the dependency upgrade absolutely needed? It will make review easier to exclude the changes about dependencies updates.